sphere-models/src/it/java/io/sphere/sdk/products/ProductProjectionSearchFilterTypesIntegrationTest.java

```diff
@@ -102,15 +105,24 @@ public static void setupProducts() {
     }
 
     @Test
-    public void filtersBooleanAttributes() throws Exception {
+    public void booleanAttributesFilters() throws Exception {
         final ProductProjectionSearch search = ProductProjectionSearch.ofStaged()
                 .withQueryFilters(model -> model.allVariants().attribute().ofBoolean(ATTR_NAME_BOOLEAN)
                         .filtered().by(true));
         assertThat(executeAndReturnIds(search)).containsOnly(product1.getId());
     }
 
+    @Ignore
     @Test
-    public void filtersTextAttributes() throws Exception {
+    public void booleanAttributesFacets() throws Exception {
+        final TermFacetExpression<ProductProjection, Boolean> facetExpr = model().allVariants().attribute().ofBoolean(ATTR_NAME_BOOLEAN)
+                .faceted().byAllTerms();
+        final ProductProjectionSearch search = ProductProjectionSearch.ofStaged().withFacets(facetExpr);
+        //assertThat(executeAndReturnTerms(search, facetExpr)).containsOnlyElementsOf(asList(TermStats.of(true, 1), TermStats.of(false, 1))); // still returning F, T. to be fixed by API
```

Comment:
The problem is not the F/T, the problem is that we don't support anything but String terms.